File: ct_206/Project 4/skip_read.py
# Opens file from provided name and allows read permissions
# Returns file object for provided file name parameter
def open_file(file_name):
  f = open(file_name, 'r', encoding='utf-8')
  return f

# Converts provided file object to tuple, then prints each element in turple
# File object is the only provided parameter
def read_from_file(f, lines_to_skip):
  data = tuple(f)
  for i, row in enumerate(data):
    if i >= int(lines_to_skip):
      print(row.replace('\n', ''))

# Closes file i/o
# File object is the only provided parameter
def close_file(f):
  f.close()


file_name = input('Please enter a file name to read from: ')
lines_to_skip = input('Please enter the number of lines to skip before showing output: ')
f = open_file(file_name)
read_from_file(f, lines_to_skip)
close_file(f)

File: ct_206/Project 4/skip_read.pl
# Obtain file to open
print "Please enter a file to open: ";
my $fileName = <STDIN>;
chomp $fileName;

# Obtain number of lines to skip
print "Please enter the number of lines you would like to skip: ";
my $linesToSkip = <STDIN>;
chomp $linesToSkip;

# Open file to read
open(f, "<$fileName") or die "Could not open file!";

# Print file data
while(<f>) {
  if ($linesToSkip > 0) {
    $linesToSkip--;
  } else {
    print $_;
  }
}

close(f1);